demos/include/aws_clientcredential.h

```diff
@@ -37,6 +37,10 @@
  * @brief Host name.
  *
  * @todo Set this to the unique name of your IoT Thing.
+ * Please note that for convenience of demonstration only we
+ * are using a #define here. In production scenarios this  
+ * should be something, the software can read as a 
```

Comment:
Misplaced comma.  Can say "In production scenarios the thing name can be something unique to the device that can be read by software, such as a production serial number, rather than a hard coded constant."

Reply:
I edited the file directly myself - so no further update required.